### PR TITLE
Adds intl-tel-input jQuery plugin

### PR DESCRIPTION
**Pull request by jafrog**
For # 328 in applications

### CODE REVIEW

**Comment by nikeshashar** [comment]
@jafrog is this on the main application form?

**Comment by sjmog** [comment]
No @nikeshashar

File: source/sass/_base.scss [hunk 1]
```diff
@@ -33,3 +34,4 @@
 @import "components/article";
 @import "components/embedded_typeform";
 @import "components/comparison_table";
+@import "components/intl_tel_input";
```
Comment (sjmog) [inline source/sass/_base.scss]: Yeah actually this makes a lot of sense 👍

**Comment by sjmog** [comment]
Awesome :shipit:

**Comment by jafrog** [comment]
@nikeshashar It's not anywhere yet, here I'm just adding a component to then add it to the applications form(s). It's a part of this ticket - https://waffle.io/makersacademy/engineering/cards/579f1068e35afaf8007075a3

**Pull request merged**